src/macaron/slsa_analyzer/checks/provenance_available_check.py

```diff
@@ -56,11 +34,8 @@ class ProvenanceAvailableFacts(CheckFacts):
     #: The primary key.
     id: Mapped[int] = mapped_column(ForeignKey("_check_facts.id"), primary_key=True)  # noqa: A003
 
-    #: The provenance asset name.
-    asset_name: Mapped[str] = mapped_column(String, nullable=True, info={"justification": JustificationType.TEXT})
-
-    #: The URL for the provenance asset.
-    asset_url: Mapped[str] = mapped_column(String, nullable=True, info={"justification": JustificationType.HREF})
```

Comment:
Please add these back as e need to store this info for provenances. Since these two columns are `nullable`, we can leave them unpopulated for now. See this [issue](https://github.com/oracle/macaron/issues/819) to add this metadata back.